src/App.test.tsx

```diff
@@ -12,17 +12,17 @@ import { ISeverity } from './model/severity'
 import { IImpactGraph } from './model/impactGraph'
 import { sendWebviewPage } from './utils/testUtils'
 import { IAnalysisStep } from './model/analysisStep'
-import { WebviewReceiveEventType } from './api'
+import { IdeEventType } from './api'
 import { LoginConnectionType, LoginProgressStatus } from './model/login'
 
 describe('App component', () => {
 	test('renders the Dependency page when the page type is "Dependency"', async () => {
 		render(<App />)
 
 		// Load Dependency.
-		const pageData = {
-			type: WebviewReceiveEventType.ShowPage,
-			pageData: {
+		const mock = {
```

Comment:
"mock" is very abstract. Please be more specific when naming variables.
```suggestion
		const mockPage = {
```
(Let's do this change in all other locations)